engine/execution/ingestion/core.go

```diff
@@ -163,19 +144,6 @@ func (e *Core) launchWorkerToExecuteBlocks(ctx irrecoverable.SignalerContext, re
 	}
 }
 
-func (e *Core) OnBlock(header *flow.Header, qc *flow.QuorumCertificate) {
```

Comment:
Since Core is just forwarding the new block to throttle, I changed to let Machine forward the block to throttle directly.